Assets/Scripts/PlayerCharacter.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class PlayerCharacter : MonoBehaviour {
+    [SerializeField]
+    private float accelerationForce = 5;
+    [SerializeField]
+    private float maxSpeed = 5;
+    [SerializeField]
+    private float jumpForce = 10;
+    [SerializeField]
+    private Rigidbody2D rb2d;
```

Comment:
Wrap all these serialized fields up in a labeled region e.g. `#region Serialized Fields`
* https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-region